discovery-provider/plugins/notifications/src/utils/redisConnection.ts

```diff
@@ -0,0 +1,18 @@
+import type { RedisClientType } from 'redis'
+import { createClient } from 'redis'
+
+const url = process.env.audius_redis_url || "redis://localhost:6379/0"
```

Comment:
I know we use snake_case in our system, but it's always bothered me bc conventionally it is UPPER_CASED and I've used the upper case in other parts of this project